test/e2e/tests/settings/4byte-directory.spec.js

```diff
@@ -27,27 +26,23 @@ describe('4byte setting', function () {
         const contractAddress = await contractRegistry.getContractAddress(
           smartContract,
         );
-        await unlockWallet(driver);
+        await logInWithBalanceValidation(driver);
 
         // deploy contract
         await openDapp(driver, contractAddress);
 
         // wait for deployed contract, calls and confirms a contract method where ETH is sent
-        await driver.delay(largeDelayMs);
```

Comment:
if we make sure we login with balance validation, we can safely remove this delay